vllm/v1/worker/gpu_model_runner.py

```diff
@@ -205,7 +205,7 @@ def __init__(
                                         pin_memory=self.pin_memory)
         self.seq_lens_np = self.seq_lens_cpu.numpy()
 
-    def _update_states(self, scheduler_output: "SchedulerOutput") -> None:
+    def _update_states(self, scheduler_output: "SchedulerOutput") -> bool:
```

Comment:
docstring, especially to explain the return type.

Reply:
Added!